src/main/java/com/example/java/test/junior/developer/UserController.java

```diff
@@ -10,20 +9,23 @@
 @RestController
 @RequestMapping("/api")
 public class UserController {
+
+    //відповідає за обробку даних, які вносить юзер
+
     ConcurrentHashMap<Long, User> users = new ConcurrentHashMap<>();
 
     @GetMapping("/{id}")
     public User getUser(@PathVariable Long id) {
         return users.get(id);
     }
 
-    @GetMapping("/")
+    @GetMapping("/users")
     public List<User> getAllUsers() {
         return new ArrayList<User>(users.values());
     }
 
-    @PostMapping("/")
-    public User getUsers(@RequestBody User user) {
+    @PostMapping("/new_user")
```

Comment:
Please, follow REST best practices. It should be just /users, not "/new_user".
Please, don't use entity as a request body. It is bad practice. Create separate DTO for request. This DTO:
- should not have id field
- should have password field